Assets/HoloToolkit-Examples/GameControllerManipulator/Example.unity

```diff
@@ -0,0 +1,518 @@
+%YAML 1.1
```

Comment:
I think a better name than just Example would be good. Like GameControllerTest.unity or something like that.